decred/decred/dcr/addrlib.py

```diff
@@ -439,7 +440,7 @@ def decodeAddressPubKey(decoded, netParams):
     if suite == STEcdsaSecp256k1:
         b = ByteArray(toAppend) + decoded[1:]
         return AddressSecpPubKey(b, netParams)
-        # return NewAddressEdwardsPubKey(decoded, netParams)
+    elif suite == STEd25519:  # nocover
         raise NotImplementedError("Edwards signatures not implemented")
```

Comment:
`raise NotImplementedError` is already excluded in `.coveragerc`, I don't think you need the `# nocover` here. It doesn't harm though.